### DISCUSSION TITLE
M3u8 mime type for GET path

### DISCUSSION THREAD

**Discussion by jcfrk101** [micronaut-projects/micronaut-core#3977] [Q&A] [answered]
I need to respond with mime type `application/x-mpegURL` or `vnd.apple.mpegURL` in order for AVPlayer on iOS to recognize the stream as an HLS stream. Is there a way to add custom mime types to micronauts? Are there any examples out there?

**Comment by ilopmar** [comment, marked as answer]
You can define any mimetype you want on your controllers:

```java
@Get(value = "/foo", produces = "application/x-mpegURL")
public foo() {
   ...
}
```

or 
```java
@Produces("application/x-mpegURL")
@Get("/foo")
public foo() {
   ...
}
```
